ecs-cli/modules/utils/compose/ecs_params_reader.go

```diff
@@ -88,3 +90,33 @@ func ReadECSParams(filename string) (*ECSParams, error) {
 
 	return ecsParams, nil
 }
+
+func ConvertToEcsNetworkConfiguration(ecsParams *ECSParams) *ecs.NetworkConfiguration {
```

Comment:
Since this file now includes more than just reading ECS Params, what do you think about renaming it to just `ecs_params.go`